pkg/rtc/mediatracksubscriptions.go

```diff
@@ -272,7 +310,18 @@ func (t *MediaTrackSubscriptions) RemoveSubscriber(participantID livekit.Partici
 	t.subscribedTracksMu.Unlock()
 
 	if subTrack != nil {
-		subTrack.DownTrack().CloseWithFlush(!willBeResumed)
+		dt := subTrack.DownTrack()
```

Comment:
can we use a function for this? duplicate code in RemoveAllSubscribers

Reply:
Thank you @cnderrauber . Added this commit to address this and also more defensive in uncaching the transceiver - https://github.com/livekit/livekit/pull/785/commits/3c0cc04c0405ba042bb370530fa42eb9c42d7113